Open Judge System/Workers/OJS.Workers.LocalWorker/LocalWorkerService.cs

```diff
@@ -108,5 +109,19 @@ private void ResetAllProcessingSubmissions()
                 }
             }
         }
+
+        /// <summary>
+        /// Creates folder in the Temp directory if not already created,
+        /// in which all strategies create their own working directories
+        /// making easier the deletion of left-over files by the background job
+        /// </summary>
+        private void CreateExecutionStrategiesDirectoryInTemp()
```

Comment:
`CreateExecutionStrategiesWorkingDirectory` would be a better name, since the fact that it's in the temp directory has little relevance.